alternator/executor.cc

```diff
@@ -4505,6 +4505,18 @@ static lw_shared_ptr<keyspace_metadata> create_keyspace_metadata(std::string_vie
     }
     auto opts = get_network_topology_options(sp, gossiper, rf);
 
+    // Tablets are not yet enabled by default on Alternator tables, because of
+    // missing support for CDC (see issue #16313). Until then, allow
```

Comment:
Because they're an experimental feature, too, and incomplete.

Reply:
When I wrote "not enabled by default", I meant not even if the tablets experimental features is turned on. Even with the tablets experimental feature, I didn't want to enable tablets on Alternator by default. By the way, the same is currently the case in CQL - it's not enough to enable the experimental feature, you need to enable it per table.